source/recipe.rst

```diff
@@ -332,6 +333,9 @@ be given for all `SynapsesClassification` elements:
   where :math:`ca` denotes the simulated calcium concentration in
   millimolar and :math:`y` a scalar such that at
   :math:`ca = 2.0:\ u_\text{final} = u`. (Markram et al., 2015)
+- `connection_model`, to specify the filename stub (without a final extension) that should
```

Comment:
It's a matter of personal taste, but I would more easily relate the concepts if this was named `synapse_model`. Please disregard if we will model connections which are not synapses

Reply:
I would prefer that we stay with the generic 'model_template' like they have specified for edges in the SONATA paper with Allen: https://github.com/AllenInstitute/sonata/blob/master/docs/SONATA_DEVELOPER_GUIDE.md#representing-edges
In NEURON and other simulators, the generic word 'model' is used for various objects: ion channels, firing behavior, synapse, gap junction, etc

Reply:
Whatever is decided, it should match what is chosen for the column in #5.  I'd prefer that we don't duplicate names in nodes and edges, simply because it makes it easier to grep for one or the other and it reduces potential confusion, but it's not a deal breaker for me.

Reply:
`connection_model` makes me think about the anatomical connection. `synapse_model` seems more appropriate to me.

Reply:
synapse_model does not work for the reason mentioned by @jamesgking - it is not only about synapses.